.github/workflows/linux-qe.yml

```diff
@@ -0,0 +1,118 @@
+name: tester-linux
+
+on:
+  workflow_run:
+    workflows: [Build Linux artifacts]
+    types:
+      - completed
+
+jobs:
+  tests:
+    runs-on: [self-hosted, linux, testing-farm]
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ['1.21']
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go }}
+      
+      - name: prepare env
+        run: |
+          sudo yum install podman openssh-server git make -y
+          ssh-keygen -t rsa -N '' -f id_rsa -q 
+      
+      - name: reserve machine from testing farm
+        env: 
+          TESTING_FARM_API_TOKEN: ${{ secrets.TESTING_FARM_API_TOKEN }}
+        run: |
+          export TESTING_FARM_API_TOKEN=$TESTING_FARM_API_TOKEN
+          eval $(ssh-agent)
+          testing-farm reserve --compose Fedora-40 --arch aarch64 --ssh-public-key id_rsa.pub --no-autoconnect | tee info
+          machine=`tail -n 1 info`
+          echo ${machine##*@} > host
+          echo "root" > username
+          #ssh -o StrictHostKeyChecking=no -i id_rsa root@$host
+     
+      - name: Download linux binary
+        uses: actions/download-artifact@v4
+        with:
+          name: linux-binary
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+
+      - name: Download qe oci image
+        id: download-qe-oci-image-artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: crc-e2e-linux-arm64
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+
+      - name: Correlate
+        env:
+          PULL_SECRET: ${{ secrets.PULL_SECRET }}
+        run: |
+          # Save pull-secret as file
+          echo "${PULL_SECRET}" > pull-secret
+
+      - name: Install CRC on host
+        run: |
+          podman run --rm -d --name crc-linux-install \
+            -e TARGET_HOST=$(cat host) \
+            -e TARGET_HOST_USERNAME=$(cat username) \
+            -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+            -e TARGET_FOLDER=crc-support \
+            -e TARGET_CLEANUP='false' \
+            -e OUTPUT_FOLDER=/data \
+            -e DEBUG='true' \
+            -v ${PWD}:/data:z \
+            -v ${PWD}/crc:/opt/crc-support/crc:z \
+            quay.io/rhqp/crc-support:v0.6-linux crc-support/install.sh crc-support/crc
+          podman logs -f crc-linux-install
```

Comment:
Can we set to WIP until we clear this usage, also when the new option to install based on assets shared this run will be different